OfficeAddin/05 Using modern JavaScript/Demos/02 Office Add-in using Angular/README.md

```diff
@@ -1,48 +1,34 @@
-# Office Add-ins: Build an Office Add-in using modern JavaScript tools and techniques - 300 Level
+## Build an Office Add-in using Angular
 
-In this demo, you will build an Office Add-in using Angular, a popular JavaScript framework.
+In this exercise, you will develop an Office Add-in using Angular and TypeScript. You will provision a new project using the Angular CLI and Office Yeoman generator, develop the add-in using Office.js, and test the add-in in Office Online.
 
 ## Running the project
```

Comment:
can you move pre-requisites before running the project please?

Reply:
Addressed across all three